test/expect-tests/chrome_trace/chrome_trace_tests.ml

```diff
@@ -0,0 +1,25 @@
+open Stdune
+open Dune_tests_common
+
+let () = init ()
+
+let buf = Buffer.create 0
+
+let time = ref 0.
+
+let c = Chrome_trace.fake time buf
+
+let () = time := 10.
+
+let () = time := 30.
+
+let () = Chrome_trace.close c
+
+let buffer_lines () = String.split_lines (Buffer.contents buf)
+
+let%expect_test _ =
+  Format.printf "%a@." Pp.to_fmt
+    (Pp.vbox (Pp.concat_map (buffer_lines ()) ~sep:Pp.cut ~f:Pp.verbatim));
+  [%expect {|
```

Comment:
Well, you removed everything from this test. Might as well remove the test?
But really, shouldn't we test something here?

Reply:
I'll add some tests that demonstrates the kind of json output we generate. It is kind of useless though, since we have no way of testing if the chrome browser understands this output.

Reply:
Done